### PR TITLE
feature2 added

### PR DESCRIPTION
**Pull request by Nimeshworkstation**
i have added two files show and hide

### CODE REVIEW

File: feature2/hide.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+i want to hide you some thing
```
Comment (smnighim) [inline feature2/hide.txt]: write clearly what you want to hide ?

File: feature2/show.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+i want to show you something
```
Comment (smnighim) [inline feature2/show.txt]: write clearly what you want to show ?

**Review by smnighim — CHANGES_REQUESTED** [review]
address the suggestions please

File: feature2/hide.txt (hunk 1 above)
Comment (Nimeshworkstation) [reply feature2/hide.txt]: i have changed the file that i wanted to hide

File: feature2/show.txt (hunk 2 above)
Comment (Nimeshworkstation) [reply feature2/show.txt]: i have changed the show file

**Review by smnighim — APPROVED** [review]
okay done

**Pull request merged**